p/themes/base-theme/template.css

```
@@ -572,6 +572,10 @@ a.btn {
 	width: 100px;
 }
 
+.header > .item.search input {
+	-webkit-appearance: none; /* csslint allow: -webkit-appearance */
```

```suggestion
	/* stylelint-disable property-no-vendor-prefix */
	-webkit-appearance: none;
	/* stylelint-enable property-no-vendor-prefix */
```

There might be a better syntax, but this one seems to work

---

I meant `-webkit-appearance: none; /* csslint allow: property-no-vendor-prefix */` btw.

Not that it matters or should be changed or anything, I just think it looks slightly neater. ;-)

---

Please send a fix if you have a better syntax @Frenzie , so that it can also be used as a reference for future needs